draft-ietf-tls-esni.md

```diff
@@ -533,6 +533,10 @@ it does a standard ClientHello, with the exception of the following rules:
    "server_name" extension.
 1. It MUST NOT include the "pre_shared_key" extension. (See
    {{flow-clienthello-malleability}}.)
+1. When the client offers the "early_data" extension in ClientHelloInner, it
+   MUST also include the "early_data" extension in ClientHelloOuter. This
+   allows servers that reject ECH and use ClientHelloOuter to safely ignore any
+   early data sent by the client per {{RFC8446}}, Section 4.2.10.
```

Comment:
This also requires pre_shared_key per s 4.2.10, so this contradicts the previous reqt.

Reply:
This is based on #414, which clarifies that the PSK must also be present. This won't land until that does.

Reply:
@ekr this should now be resolved give that #414 landed. Can you please take another look?